index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="css/mobileBgProg.css">
+    <link rel="stylesheet" media="screen and (max-width: 768px)" href="./css/mobile.css">
+    <link rel="stylesheet" href="./css/common.css">
+    <link rel="stylesheet" media="screen and (max-width: 768px)" href="./css/hamburger.css">
+    <link rel="stylesheet" media="screen and (min-width: 769px)" href="./css/desktopCom.css">
+
+</head>
+<body>
+    <header class="header">
+        <div class="desktop-header">
+            <div class="dh-container">
+                <img src="./assets/social/facebook_f_64px.png" alt="">
+                <img src="./assets/social/twitter_64px.png" alt="">
```

Comment:
- [ ] Kindly add alt attributes to all your image tags so that it can be accessible by other devices.